source/assets/sass/components/_callouts.scss

```diff
@@ -31,9 +31,9 @@
   }
 
   &--impl-status {
-    font-size: 0.8em;
+    font-size: var(--status-text-size);
     margin-top: var(--gutter-negative);
-    padding: 0.8rem;
+    padding: var(--gutter-minus);
```

Comment:
I know these values aren't he same here, but they are small enough changes that they aren't noticeable

Reply:
these different callouts variations have different padding settings. could we use a custom property to reset some of those too?

Reply:
Probably. I didn't redo the structure/strategy of most of this PR. Tried to keep it narrow otherwise I would tend to go too far down the rabbit hole on this one PR. I will make a checklist item for this as I think this could be expanded in more areas of this CSS